docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -162,14 +164,14 @@ Operations Involved in Masking
 There are 2 operations to mask a detector and thus spectrum related
 
 1. Set the detector in workspace's instrument's *parameter map* to *masked*.
-2. Clear the data associated with the spectrum with detectors that are masked.
+2. Zero the data associated with the spectrum with detectors that are masked.
```

Comment:
Thanks for the clarification! Unfortunately this PR is already merged, but I have opened an issue to fix this: #18407